src/components/latex-textarea/LatexTextArea.module.less

```diff
@@ -0,0 +1,9 @@
+@import '@variables';
+
+.root {
```

Comment:
call it something like `.texEditorTextArea`

Reply:
Renamed to `.latexTextArea`. I renamed `.root` to `.pdfViewer` in `PdfViewer` as well.